v1.0.0.yaml

```diff
@@ -2809,131 +2722,88 @@ paths:
-                  type: boolean
-                  description: |
-                    Default is `true`. If set to `true` and a file already exists at the exact location, its AITags will be removed. Set `overwriteAITags` to `false` to preserve `AITags`.
-                overwriteTags:
-                  type: boolean
-                  description: |
-                    Default is `true`. If the request does not have `tags`, `overwriteTags` is set to `true` and a file already exists at the exact location, exiting `tags` will be removed. In case the request body has `tags`, setting `overwriteTag`s to `false` has no effect and request's tags are set on the asset.
-                overwriteCustomMetadata:
-                  type: boolean
-                  description: |
-                    Default is `true`. If the request does not have `customMetadata`, `overwriteCustomMetadata` is set to `true` and a file already exists at the exact location, exiting `customMetadata` will be removed. In case the request body has `customMetadata`, setting `overwriteCustomMetadata` to `false` has no effect and request's customMetadata is set on the asset.
-                customMetadata:
-                  type: object
-                  description: |
-                    Stringified JSON key-value data to be associated with the asset. Checkout `overwriteCustomMetadata` parameter to understand default behaviour. Before setting any custom metadata on an asset you have to create the field using custom metadata fields API.
+            Learn [how to implement client-side file upload](#how-to-implement-client-side-file-upload).
+      requestBody:
+       content:
+          multipart/form-data:
+            schema:
+              oneOf:
+                - $ref: '#/components/request/ServerSideUploadV1'
+                - $ref: '#/components/request/ClientSideUploadV1'
       responses:
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          $ref: '#/components/responses/Forbidden'
         "200":
           description: File uploaded successfully.
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/uploadResponse"
+                $ref: "#/components/responses/Upload"
+        "202":
+          description: File accepted for upload. Pre transformation for video is being processed.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: The pre-transformation has been queued successfully.
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    examples:
```

Comment:
Note: The syntax for the examples field is incorrect. Multiple examples can only be added by creating a separate property for each one of them. See https://swagger.io/docs/specification/adding-examples/ for more details.

Only the first example will be visible otherwise and the rest won't appear in API tools like Stoplight.

> You might also need to move `examples` under `application/json` instead of keeping it under `schema.properties.*`

Reply:
ye dikh rahe hai. and hamare docs me bhi dikh rahe hai.

<img width="500" alt="image" src="https://github.com/imagekitio/openapi/assets/45416977/f6bd2b91-5b7c-48e0-a125-1ca6257b5997">